docs/guides/languages.md

```diff
@@ -153,7 +153,7 @@ Language Codes
 --------------
 The following is a list of official language codes.
 
-**NOTE:** For supported language translations, please see the [Languages Folder (/lang)](../../lang) folder located in the project root.
```

Comment:
These docs don't really live on github anymore -they get copied to the/docs
repo and deployed on docs.videojs.com, so the links have to be fixed for
that.

On Sun, Jan 3, 2016 at 19:20 Thijs Triemstra <notifications@github.com>
wrote:

> In docs/guides/languages.md
> <https://github.com/videojs/video.js/pull/2965#discussion_r48699395>:
>
> > @@ -153,7 +153,7 @@ Language Codes
> >  --------------
> >  The following is a list of official language codes.
> >
> > -**NOTE:** For supported language translations, please see the [Languages Folder (/lang)](../../lang) folder located in the project root.
>
> This relative URL seems to work fine on Github, where's the problem?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/videojs/video.js/pull/2965/files#r48699395>.
>
-- 
Robert Crooks
Director of Learning Services
Brightcove, Inc